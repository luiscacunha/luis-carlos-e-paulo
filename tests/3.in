RP Medicina Alice
RP Medicina Bob
RP Enfermagem Bob
RP Enfermagem Trudy
RP Auxiliar Phill
RP Auxiliar James
RU Bill Adulto
RU Brad Jovem
RU Carl Idoso
RU Claire Adulto
RF Smith
RF Jones
AF Bob Smith
AF Claire Smith
AF Bill Jones
AF Carl Jones
MC Bill
Consulta
Medicina Alice
PequenaCirurgia
Medicina Bob
Enfermagem Trudy
Auxiliar James
Enfermagem
Auxiliar Phill
Enfermagem Bob
Consulta
Medicina Alice

MC Carl
Enfermagem
Enfermagem Trudy
Enfermagem Bob
Consulta
Medicina Alice

LCF Smiths
LCF Smith
LCF Jones
LSP Medicina Halice
LSP Medicina Alice
LMS PequenaCirurgia
LMS Medicina
LMS Enfermagem

