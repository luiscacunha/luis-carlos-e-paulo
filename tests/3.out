Profissional registado com sucesso.
Profissional registado com sucesso.
Profissional registado com sucesso.
Profissional registado com sucesso.
Profissional registado com sucesso.
Profissional registado com sucesso.
Utente registado com sucesso.
Utente registado com sucesso.
Utente registado com sucesso.
Utente registado com sucesso.
Família registada com sucesso.
Família registada com sucesso.
Utente inexistente.
Utente associado a família.
Utente associado a família.
Utente associado a família.
Cuidados marcados com sucesso.
Cuidados marcados com sucesso.
Família inexistente.
Família sem cuidados de saúde marcados.
Bill Consulta Medicina Alice.
Bill Consulta Medicina Alice.
Bill PequenaCirurgia Medicina Bob.
Bill PequenaCirurgia Enfermagem Trudy.
Bill PequenaCirurgia Auxiliar James.
Bill Enfermagem Enfermagem Bob.
Bill Enfermagem Auxiliar Phill.
Carl Consulta Medicina Alice.
Carl Enfermagem Enfermagem Bob.
Carl Enfermagem Enfermagem Trudy.
Profissional de saúde inexistente.
Consulta Bill.
Consulta Bill.
Consulta Carl.
Medicina Bob Bill.
Enfermagem Trudy Bill.
Auxiliar James Bill.
Serviço inexistente.
Enfermagem Bob Bill.
Auxiliar Phill Bill.
Enfermagem Bob Carl.
Enfermagem Trudy Carl.
